jwst/resample/resample.py

```diff
@@ -181,6 +183,16 @@ def update_fits_wcs(self, model):
         """
         Update FITS WCS keywords of the resampled image.
         """
+        # Delete any SIP-related keywords first
+        pattern = r"(cd[123]_[123]|ap?_|bp?_)"
+        regex = re.compile(pattern)
+        # Make a copy so we don't delete keys in the model while iterating
+        wcsinfo = model.meta.wcsinfo.instance.copy()
```

Comment:
(per our slack convo) change this to list(model.meta.wcsinfo.instance.keys()) instead of copying the whole thing.